docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_preventing_redundant_rebuilds.adoc

```diff
@@ -0,0 +1,60 @@
+= Preventing redundant rebuilds
+
+After you upgrade the build pipeline for a component, {ProductName} rebuilds the component when new pull requests (PRs) include changes to that component. This behavior helps you identify any issues the PR might cause before merging it. 
+
+By default, {ProductName} configures upgraded pipelines to rebuild a component _only_ if a PR includes changes to that component. This configuration prevents redundant rebuilds in repositories that contain multiple components. 
+
+However, depending on the structure of your repository and the dependencies of your component, you may want to adjust the default configuration for rebuilds to better suit your needs. In particular, the default configuration might still rebuild a component too often. You can change this behavior by editing the `on-cel-expression` annotation.   
```

Comment:
This is not displaying correctly in the output -  `on-cel-expression`

Reply:
Hmm for me it is correctly displaying. Is it possible there's a browser issue on your end?